examples/minimal/README.md

```diff
@@ -0,0 +1,3 @@
+# Minimal MUD example project with multiple clients
+
+This example project demonstrates how to use the MUD framework to create a simple on-chain application. It includes several client types that you can pick from to run the app or get inspiration from, all using the same underlying smart contracts.
```

Comment:
```suggestion
This example project demonstrates how to use the MUD framework to create a simple on-chain application. It includes several client types that you can pick from to run the app or get inspiration from, all using the same underlying smart contracts.

## Getting started
1. Since this project uses local MUD dependencies, you need to prepare them first by running this command in the MUD root directory: `pnpm install && pnpm build`
2. To start the development environment of this project, run `pnpm dev` in the project root.
3. Open [localhost:3000](http://localhost:3000) in your browser.
```